gammapy/cube/tests/test_simulate.py

```diff
@@ -182,10 +182,40 @@ def test_mde_run(dataset):
     sampler = MapDatasetEventSampler(random_state=0)
     events = sampler.run(dataset=dataset, observation=obs)
 
-    assert len(events.table) == 2349
-    assert_allclose(events.table["ENERGY"][0], 1.894698, rtol=1e-5)
-    assert_allclose(events.table["RA"][0], 266.454448, rtol=1e-5)
-    assert_allclose(events.table["DEC"][0], -30.870316, rtol=1e-5)
+    assert len(events.table) == 2422
```

Comment:
Why did those test values change? It seems unrelated somehow...

Reply:
Yeah I know, it's the same thing I was asking myself. They shouldn't change but they did.

Reply:
I think they changed because the order in which the sample methods are called changed. So the seed for the random call, might be a different one...